lib/features/payment/payment_details_page.dart

```diff
@@ -29,14 +27,22 @@ class PaymentDetailsPage extends HookConsumerWidget {
     super.key,
   });
 
+  PaymentMethods<dynamic> _getPaymentMethod(PaymentState paymentState) {
+    if (paymentState.transactionType == TransactionType.deposit) {
+      return PayinMethods(paymentState.payinMethods);
+    } else {
+      return PayoutMethods(paymentState.payoutMethods);
+    }
+  }
+
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final selectedPaymentMethod = useState<Object?>(null);
+    final selectedPaymentMethod = useState<dynamic>(null);
```

Comment:
going from `Object?` to `dynamic` doesn't seem like a huge upgrade.. not easy to make `PayinMethod` and `PayoutMethod` play nice together